libs/designsystem/src/lib/components/button/button.component.spec.ts

```diff
@@ -313,3 +314,29 @@ describe('ButtonComponent', () => {
     });
   });
 });
+
+describe('ButtonComponent using an ngIf directive', () => {
```

Comment:
Consider moving to [existing button integration test](https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/button/button.component.integration.spec.ts#L466)

Reply:
Should be resolved by new commits.